features/bootstrap/SecurityContext.php

```diff
@@ -43,17 +43,24 @@ private function logAs(UserInterface $user): void
     {
         $driver = $this->getSession()->getDriver();
         $session = $this->getContainer()->get('session');
+
+        if ($driver instanceof Selenium2Driver) {
+            $page = $this->getSession()->getPage();
+
+            $this->visitPath('/connexion');
+            $page->findField('_adherent_email')->setValue($user->getUsername());
+            $page->findField('_adherent_password')->setValue(\AppBundle\DataFixtures\ORM\LoadAdherentData::DEFAULT_PASSWORD);
```

Comment:
any reason not to declare a `use` statement for `LoadAdherentData` ?

Reply:
no